### DISCUSSION TITLE
Winlogbeat via Forward Node?

### DISCUSSION THREAD

**Discussion by wildlyunder** [Security-Onion-Solutions/securityonion#6018] [Unsupported Versions] [answered]
Hi,

Is it possible to send Winlogbeat via a Forward Node? If not, what's the best way to get the Winlogbeats logs from one security zone to the next security zone with the manager?

I want to use a light VM to do this... I was thinking about using a standalone logstash server if it's not possible to use a forward node.

**Comment by weslambert** [comment, marked as answer]
It is not currently possible to point Winlogbeat to a forward node. You might consider a separate dedicated/locked down interface if you don't want to point to the other manager interface. Otherwise, you might have to go with an intermediary instance as you've mentioned.

**Comment by PatRussell87** [comment]
I am working on two ways to deal with the same issue.

One is to create a Windows Event Collector in the security zone you want to read events from. Forward the logs from your endpoints to the collector, then have the collector send out to your manager. I have this working in one environment, requiring only the event collector to pass through the firewall.

The second option is to switch to using Wazuh for your event collecting. You can send Wazuh traffic to the forward node, which will then send to your manager.

So far, I prefer the event collector route as it seems to parse the logs better in Security Onion.
